File: VB/MyWizardControl/MyWizardAeroModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraWizard

Namespace WindowsApplication1
	Public Class MyWizardAeroModel
		Inherits WizardViewInfo.WizardAeroModel
		Public Sub New(ByVal viewInfo As WizardViewInfo)
			MyBase.New(viewInfo)

		End Sub


		Public Overrides Function GetTitleBarTitleTextBounds(ByVal iconVisible As Boolean) As Rectangle
			Return Rectangle.Empty

		End Function


		Public Overrides Sub UpdateButtonsLocation()
			MyBase.UpdateButtonsLocation()
			TryCast(ViewInfo, MyWizardViewInfo).NextButton.Bounds = GetNextButtonBounds()
		End Sub

		Private Function GetNextButtonBounds() As Rectangle
			Dim backButtonBounds As Rectangle = GetBackButtonBounds()
			backButtonBounds.Offset(backButtonBounds.Width, 0)
			Return backButtonBounds
		End Function

	End Class
End Namespace

File: CS/MyWizardControl/MyWizardAeroModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraWizard;

namespace WindowsApplication1
{
    public class MyWizardAeroModel : WizardViewInfo.WizardAeroModel
    {
        public MyWizardAeroModel(WizardViewInfo viewInfo)
            : base(viewInfo)
        {

        }


        public override Rectangle GetTitleBarTitleTextBounds(bool iconVisible)
        {
            return Rectangle.Empty; ;
        }


        public override void UpdateButtonsLocation()
        {
            base.UpdateButtonsLocation();
            (ViewInfo as MyWizardViewInfo).NextButton.Bounds = GetNextButtonBounds();
        }

        private Rectangle GetNextButtonBounds()
        {
            Rectangle backButtonBounds = GetBackButtonBounds();
            backButtonBounds.Offset(backButtonBounds.Width, 0);
            return backButtonBounds;
        }

    }
}
